core/common/src/thrift/file_system_master.thrift

```diff
@@ -123,6 +123,27 @@ struct MountTOptions {
 }
 struct MountTResponse {}
 
+struct GetMountTableTResponse {}
+
+/*
+struct GetMountTableTResponse {
+  1: map<string, MountPointInfo> mountTable
+}
+
+struct MountPointInfo {
```

Comment:
You should change `core/common/src/main/java/alluxio/wire/MountPointInfo.java` to add two new methods `toThrift`. you can check how `FileInfo.java` in the same dir does this.